apps/builder/app/shared/copy-paste/plugin-instance.ts

```diff
@@ -184,24 +184,16 @@ export const onPaste = (clipboardData: string) => {
         dropTarget
       );
 
-      // find all local style sources of copied instances
-      const newStyleSourceIds = findSubtreeLocalStyleSources(
-        new Set(copiedInstanceIds.values()),
-        new Map(
-          data.styleSources.map((styleSource) => [styleSource.id, styleSource])
-        ),
-        new Map(
-          data.styleSourceSelections.map((styleSourceSelection) => [
-            styleSourceSelection.instanceId,
-            styleSourceSelection,
-          ])
-        )
+      const localStyleSourceIds = findLocalStyleSourcesWithinInstances(
+        data.styleSources,
+        data.styleSourceSelections,
+        new Set(copiedInstanceIds.keys())
```

Comment:
So the bug was that we called with `copiedInstanceIds.values()` before?

Reply:
Yep, I overlooked the case